python/cugraph/__init__.py

```diff
@@ -31,6 +31,8 @@
 
 from cugraph.snmg.link_analysis.mg_pagerank import mg_pagerank
 
+from cugraph.proro.components import strong_connected_component
```

Comment:
is proro a typo? (shouldn't this be proto?)

Reply:
:(  fixing that now